web/libs/ui/src/index.ts

```diff
@@ -1,4 +1,6 @@
 export * from "./lib/checkbox/checkbox";
 export * from "./lib/Toast/Toast";
+export * from "./lib/label/label";
+export * from "./lib/toggle/toggle";
```

Comment:
still the case is wrong

Reply:
it matches the current file names - i'll be standardizing the file name casing in another ticket

Reply:
Let's discuss this more in the frontend guild, but not hold any PR to a standard that we haven't ratified yet.